src/components/portal/payment/Payment.js

```diff
@@ -25,12 +25,17 @@ class Payment extends Component {
     this.onError = this.onError.bind(this);
   }
 
+
   componentDidMount() {
     let { cart, cart_products } = this.props.product;
     if (isEmpty(cart) || isEmpty(cart_products)) {
       this.props.history.push('/');
     }
   }
+  // forceUpdate()
+  // {
+  //   this.totalPrice = 0;
+  // }
```

Comment:
do we need to keep this ?

Reply:
not needed. removed those lines now, thank you.